examples/source/1.components/amp-sidebar.html

```diff
@@ -42,102 +44,178 @@
     }
     .sample-sidebar li,
     nav[toolbar] li {
-      margin-bottom: var(--space-2);
-      margin-left: 0;
-      margin-right: var(--space-2);
+      margin: var(--space-2);
+      margin-left: var(--space-4);
       list-style: none;
     }
+    amp-sidebar ul, nav[toolbar] ul {
+      display: block;
+      padding-left: 0;
+    }
     .previewOnly {
       margin: var(--space-2);
     }
     #sidebar-right nav.amp-sidebar-toolbar-target-shown {
         display: none;
     }
+    button {
+      margin-left: var(--space-2);
+    }
+    header {
+      background-color: lightgray;
+      padding-bottom: var(--space-2);
+    }
+    .hamburger {
+      top: 0;
+      right: 0;
+      position: absolute;
+    }
+    h1 {
+      padding: var(--space-1);
+      margin: 0;
+      font-size: 2em;
+    }
+    .desktop-sidebar {
+      background-color: lightblue;
+    }
+    @media (min-width: 783px) {
+      .hamburger {
+        display: none;
+      }
+      main {
+        display: flex;
+        flex-direction: row;
+      }
+      aside {
+        width: 300px;
+      }
+    }
   </style>
 </head>
 <body>
+  <!-- Header -->
```

Comment:
Can you move these below into the Desktop Toolbar section? Otherwise they will be displayed at the top of the example page where it's a bit confusing:

<img width="828" alt="Screen Shot 2020-03-19 at 10 35 38" src="https://user-images.githubusercontent.com/380472/77052632-67dd7400-69cd-11ea-8ac2-10e5e9ebf124.png">

Reply:
Thanks!  I have removed the comments so these do not need to be annotated in the documentation.